sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -0,0 +1,1299 @@
+## API Report File for "@azure/ai-form-recognizer"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference types="node" />
+
+import { AzureKeyCredential } from '@azure/core-auth';
+import { KeyCredential } from '@azure/core-auth';
+import { OperationOptions } from '@azure/core-client';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PipelineOptions } from '@azure/core-rest-pipeline';
+import { PollerLike } from '@azure/core-lro';
+import { PollOperationState } from '@azure/core-lro';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export type AnalysisPoller<Result = AnalyzeResult<AnalyzedDocument>> = PollerLike<DocumentAnalysisPollOperationState<Result>, Result>;
+
+// @public
+export interface AnalyzedDocument {
+    boundingRegions?: BoundingRegion[];
+    confidence: number;
+    docType: string;
```

Comment:
I am not sure if the abbreviation is justified in this case. Did you consider `type` or `documentType`?

Reply:
This is the name from the service schema. We might change this to documentType in the future (or maybe even to "kind" for TypeScript idiomicity -- but document types are a core service concept so that seems unlikely), but for now it's docType to match with other FR tools such as the training/labeling tool.